### ISSUE TITLE
question about dL_dviewmatrix and dL_dprojmatrix in RasterizeGaussiansBackwardCUDA

### ISSUE THREAD

**Issue by Howie-Ye** [taekkii/deblurgs#2]
Thank you for your great work. 
I have observed some modifications in the backward part that were not mentioned in the paper. 
As I am utilizing a different version of the diff-gaussian-rasterization. Without these modifications, the grad of _nu,_rot and _trans are None after loss.backward(). 
I would greatly appreciate it if you could provide the derivations for dL_dviewmatrix and dL_dprojmatrix.
I am looking forward to your response.
